pages/task/read/views/project.jsx

```diff
@@ -75,14 +76,42 @@ const Project = ({ task, project, establishment, children, schema, formFields })
             {
               task.status === 'with-inspectorate' && (
                 <Fragment>
-                  <p><Snippet>declarations.pel-holder</Snippet></p>
-                  <p><strong>{declarationAnswer(taskMeta.authority)}</strong></p>
-
-                  <p><Snippet>declarations.awerb</Snippet></p>
-                  <p><strong>{declarationAnswer(taskMeta.awerb)}</strong></p>
-
-                  <p><Snippet>declarations.ready-for-inspector</Snippet></p>
-                  <p><strong>{declarationAnswer(taskMeta.ready)}</strong></p>
+                  <p><strong><Snippet>declarations.pel-holder.question</Snippet></strong></p>
+                  <p>{declarations.authority}</p>
+                  {
+                    declarationConfirmed(declarations.authority) &&
+                      <Fragment>
+                        <p>
+                          <Snippet>declarations.pel-holder.name</Snippet> {declarations['authority-pelholder-name']}
+                        </p>
+                        <p>
+                          <Snippet>declarations.pel-holder.endorsement-date</Snippet> {declarations['authority-endorsement-date']}
+                        </p>
+                      </Fragment>
+                  }
+
+                  <p><strong><Snippet>declarations.awerb.question</Snippet></strong></p>
+                  <p>{declarations.awerb}</p>
+                  {
+                    declarationConfirmed(declarations.awerb) &&
+                      <Fragment>
+                        <p>
+                          <Snippet>declarations.awerb.review-date</Snippet> {declarations['awerb-review-date']}
+                        </p>
+                      </Fragment>
+                  }
+                  {
+                    // we don't collect a reason for 'Not yet'
+                    declarations.awerb.toLowerCase() === 'no' &&
```

Comment:
```suggestion
                    !declarationConfirmed(declarations.awerb) &&
```

Reply:
Oh wait, there's _three_ options now?

Reply:
Yeah, that's why it's like that and not using the function